### ISSUE TITLE
replace business insider figure

### ISSUE THREAD

**Issue by mellybelly** [flattenthecurve/guide#47]
I prefer this one, because it has *really* made the difference in explaining to elderly or skeptical people about what is different with COVID-19.

https://www.businessinsider.com/coronavirus-compared-to-flu-mortality-rates-2020-3

**Comment by rousik** [comment]
I agree that full side-by-side comparison of the two (mortality of flu vs covid-19 for various age ranges)  would be much more to the point and easy to quickly understand. I'm going to see how I can highlight this issue w/ our content team and if they agree with this change I can take care of it.
